src/main/java/helpers/Helpers.java

```diff
@@ -0,0 +1,20 @@
+package helpers;
+
+public class Helpers {
```

Comment:
Although it may not be necessary, but perhaps can consider to include comment/brief description of `Helpers` class?

Reply:
Yes I think it would be good to have a description too. Will add in future commits.